packages/dom/src/serialize-cssom.js

```diff
@@ -34,6 +42,47 @@ export function serializeCSSOM({ dom, clone }) {
       cloneOwnerNode.remove();
     }
   }
+
+  // clone stylesheets in shadowRoot
+  // https://github.com/WICG/construct-stylesheets/issues/93
+  if (!adoptedStylesMap) {
+    adoptedStylesMap = new Map();
+  }
+
+  for (let sheet of dom.adoptedStyleSheets) {
+    const styleLink = document.createElement('link');
+    styleLink.setAttribute('rel', 'stylesheet');
+
+    if (!adoptedStylesMap.has(sheet)) {
```

Comment:
* can sheet be used as a key? (Not sure if a hash function is implemented for sheet)
* add a test with multiple adopted sheets of same & different content to verify.

Reply:
Yup, have already added a test here - https://github.com/percy/cli/blob/d4ff7c7256fa342b0d57a3014fcc24b9ff841c25/packages/dom/test/serialize-css.test.js#L97-L136